generators/add-web-assets/exc-react/templates/src/App.js

```diff
@@ -41,6 +41,25 @@ export default class App extends React.Component {
 
     console.log('runtime object:', this.props.runtime)
     console.log('ims object:', this.props.ims)
+
+    // respond to configuration change events (e.g. user switches org)
+    this.props.runtime.on('configuration', ({ imsOrg, imsToken, locale }) => {
+      console.log('configuration change', { imsOrg, imsToken, locale })
+      // if (imsOrg !== this.state.imsOrg) {
+      // /* org change */
+      // }
+      // this.setState({ imsOrg, imsToken, locale })
+    })
+
+    // respond to history change events
+    this.props.runtime.on('history', ({ type, path }) => {
+      console.log('history change', { type, path })
+      const cleanedPath = path[0] === '/' ? path : '/' + path
+      if (type === 'external' && this.props.runtime.history.location.pathname !== cleanedPath) {
+        // this.props.runtime.history.replace(cleanedPath)
```

Comment:
delete commented out code?

Reply:
purpose of this conditional now that the comments are commented out?

Reply:
the purpose of the comment is to show how an example of how to use the handler, but yes in that case I should also comment out the conditional

Reply:
d'oh! forgot this was a generator.... 🤦